scripts/biomedical/mesh/unit-tests/mesh_descriptor_test.csv

```diff
@@ -0,0 +1,16 @@
+DescriptorID,DescriptorName,TreeNumber,NLMClassificationNumber,DateCreated,DateRevised,DateEstablished,Descriptor_dcid,DescriptorParentID
```

Comment:
Ideally, these should be one column for Descriptor_ID. If there are multiple values for any given property, then they should be in a list. Right now there is a bunch of duplicate information and in many cases straight-up duplicate lines in the output files. Please remove.